extensions/scarb-doc/tests/data/json_reexports.json

```diff
@@ -130,7 +130,7 @@
                   "item_data": {
                     "name": "TestStruct",
                     "doc": null,
-                    "signature": "pub struct TestStruct {\n    abc: u32,\n}",
+                    "signature": "pub struct TestStruct {}",
```

Comment:
I see that Rust adds a comment marking omitted private items, like this:
```
pub struct TestStruct {/* private fields */}
```
(e.g. https://docs.rs/cairo-lang-macro/latest/cairo_lang_macro/struct.AuxData.html) 
Should we do smth similar? 🤔